src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -81,6 +81,10 @@ function SidebarLinksData({isFocused, allReportActions, betas, chatReports, curr
         return reportIDsRef.current || [];
     }, [allReportActions, betas, chatReports, currentReportID, policies, priorityMode, isLoading]);
 
+    const currentReportIDRef = useRef(currentReportID);
+    currentReportIDRef.current = currentReportID;
+    const isActiveReport = useCallback((reportID) => currentReportIDRef.current === reportID, []);
```

Comment:
Can't we simplify the block like this?

const isActiveReport = useCallback((reportID) => currentReportID === reportID, [currentReportID]);